### PR TITLE
Fix crash

### PR DESCRIPTION
**Pull request by ntnmrndn**

### CODE REVIEW

File: Sources/MosaiqueAssetsPicker/Views/AssetDetailCell.swift [hunk 1]
```diff
@@ -94,7 +94,7 @@ final class AssetDetailCell: UICollectionViewCell, AssetDetailCellBindable {
     
     public func updateSelection(isItemSelected: Bool) {
         if isItemSelected {
-            imageView.layer.borderColor = configuration.selectionColor.cgColor
+            imageView.layer.borderColor = configuration?.selectionColor.cgColor
```
Comment (muukii) [inline Sources/MosaiqueAssetsPicker/Views/AssetDetailCell.swift:97]: I can understand it fixes that crash, however, it seems that just suppress runtime-error.
With this, we will miss silently error.
`configuration` seems it's declared as guaranteed non-nill  by developer

So the actual root cause might be outside this cell.

**Comment by ntnmrndn** [comment]
```

**Comment by ntnmrndn** [comment]
It seems that `isSelected` is set during the `dequeueReusableCellWithReuseIdentifier ` call  (and before we could set the configuration)

**Comment by muukii** [comment]
@ntnmrndn @JohnEstropia 
Thanks, I have something idea to clean the code up.
Would you mind if I submit a PR as another idea later?

**Comment by ntnmrndn** [comment]
Sure !

**Pull request closed without merging**